setup.py

```diff
@@ -428,16 +228,14 @@ def standardsetup(name, version, custompackages=[], customdatafiles=[]):
         packages.append("translate.%s" % subpackage)
 
     datafiles = getdatafiles()
-    dosetup(name, version, packages + custompackages, datafiles + customdatafiles,
-            translatescripts + translatebashscripts)
+    console_scripts = build_console_scripts(translatescripts)
```

Comment:
Please ensure that `translatescripts` list all the commands that you have removed. For example `idml2po` is missing.